README.md

```diff
@@ -1,6 +1,11 @@
 # DevOps-Apprenticeship-Notes
 
 
+* https://www.youtube.com/watch?v=0yWAtQ6wYNM
+* Hands-On Tutorials for Amazon Web Services (AWS)
+
+
+
```

Comment:
I dont think we need this much whitespace within the markdown. Markdown, by default, removes the extra whilespace. 

This is not a blocker, just thought I would mention it.